src/service/file.js

```diff
@@ -13,18 +13,23 @@ export const uploadFile = async (base64String = '') =>
   });
 
 export const uploadPhoto = async (formData) => {
-  const result = await api.post('/upload/photo', formData, {
-    headers: {
-      'Content-Type': 'multipart/form-data'
-    },
-    testing: 'test'
-  });
+  try {
+    const result = await api.post('/upload/photo', formData, {
+      headers: {
+        'Content-Type': 'multipart/form-data'
+      },
+      testing: 'test'
+    });
 
-  if (result.data) {
-    return result.data;
+    if (result.data) {
+      return result.data;
+    }
+  } catch (e) {
+    if (__DEV__) {
+      console.log('uploadPhoto:', e.response);
+    }
+    return null;
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The error handling in the `uploadPhoto` function has been improved by adding a try-catch block. However, it's important to note that the catch block only logs the error if the app is in development mode (`__DEV__`). In production, this error will be swallowed and may make debugging difficult. Consider using a logging service to capture these errors in production as well.

Also, the function now returns `null` when an error occurs. This change could potentially break any code that calls this function and expects a non-null return value. Make sure all calling code can handle a null return value.

```diff
  export const uploadPhoto = async (formData) => {
  try {
    const result = await api.post('/upload/photo', formData, {
      headers: {
        'Content-Type': 'multipart/form-data'
      },
      testing: 'test'
    });
 
    if (result.data) {
      return result.data;
    }
  } catch (e) {
    // Consider using a logging service for production
    if (__DEV__) {
      console.log('uploadPhoto:', e.response);
    }
    return null; // Ensure calling code can handle null return value
   }
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->